WalletWasabi.Backend/Global.cs

```diff
@@ -113,7 +113,7 @@ public async Task InitializeAsync(CoordinatorRoundConfig roundConfig, Cancellati
 
 				var coinVerifierApiClient = new CoinVerifierApiClient(CoordinatorParameters.RuntimeCoordinatorConfig.CoinVerifierApiAuthToken, RpcClient.Network, HttpClient);
 				var whitelist = await Whitelist.CreateAndLoadFromFileAsync(CoordinatorParameters.WhitelistFilePath, cancel).ConfigureAwait(false);
-				coinVerifier = new(CoinJoinIdStore, coinVerifierApiClient, whitelist, CoordinatorParameters.RuntimeCoordinatorConfig);
+				coinVerifier = new(CoinJoinIdStore, coinVerifierApiClient, whitelist, CoordinatorParameters.RuntimeCoordinatorConfig, Path.Combine(CoordinatorParameters.CoordinatorDataDir, "CoinVerifierAudits"));
```

Comment:
```suggestion
				coinVerifier = new(CoinJoinIdStore, coinVerifierApiClient, whitelist, CoordinatorParameters.RuntimeCoordinatorConfig, archiverDirectoryPath: Path.Combine(CoordinatorParameters.CoordinatorDataDir, "CoinVerifierAudits"));
```

though "audits" and "archiver directory path" sound like two different things to me

Reply:
Hmm, yes `auditsDirectoryPath` would be a better param name IMO.